zedd-app/src/components/Calendar.tsx

```diff
@@ -222,32 +265,66 @@ const CalendarBase = <T extends Interval>({
     [viewportXYToTime, local, onSliceStartChange, onSliceEndChange],
   )
   const dragStop = useCallback(() => {
+    if (local.currentPositionValid) {
+      onSliceStartEndChange(local.currentDraggingSlice!, local.currentDragPosition!)
+    } else {
+      if (local.correctSlicePosition) {
+        onSliceStartEndChange(local.currentDraggingSlice!, local.correctSlicePosition.interval!)
+      } else {
+        onSliceStartEndChange(local.currentDraggingSlice!, {
+          start: local.dragSliceStart,
+          end: local.dragSliceEnd,
+        })
```

Comment:
local.dragSliceStart/End as well as this block should be unnecessary. The assigned value is the same as the current value.